Makefile

```diff
@@ -11,11 +11,13 @@ GCC = g++
 endif
 LD = $(GCC)
 
-ifndef RELEASE
-CFLAGS += -Werror -Wimplicit-fallthrough
+ifdef LLD
```

Comment:
Why can't lld it be on by default in non-release mode?

Reply:
Have you tried compiling it with newest clang? I got following errors with clang 6.0:
>>> defined in /usr/lib/x86_64-linux-gnu/libopenal.so
>>> referenced by audio_device.cpp:17
>>>               obj-opt/audio_device.o:(_ZN11AudioDevice10initializeB5cxx11Ev)
/usr/lib/llvm-6.0/bin/ld.lld: error: cannot preempt symbol: alcCreateContext
>>> defined in /usr/lib/x86_64-linux-gnu/libopenal.so
>>> referenced by audio_device.cpp:18
>>>               obj-opt/audio_device.o:(_ZN11AudioDevice10initializeB5cxx11Ev)
...